function_builder.go

```diff
@@ -1,33 +0,0 @@
-package function
-
-import (
-	"fmt"
-)
-
-const (
-	BuilderPack    = "pack"
-	BuilderS2i     = "s2i"
-	DefaultBuilder = BuilderPack
-)
-
-func AllBuilders() []string {
-	return []string{BuilderPack, BuilderS2i}
-}
-
-// ErrInvalidBuilder indicates that the passed builder was invalid.
-type ErrInvalidBuilder error
-
-// ValidateBuilder validatest that the input Build type is valid for deploy command
-func ValidateBuilder(builder string) error {
```

Comment:
validator is now in `main`, as that is the only place that knows the final bounded set